superbench/benchmarks/micro_benchmarks/nvbandwidth.py

```diff
@@ -111,6 +111,10 @@ def _preprocess(self):
 
         if self._args.disable_affinity:
             command += ' --disableAffinity'
+            rank = int(os.getenv('PROC_RANK', '0'))
+            if os.getenv('NUMA_NODES'):
+                numa = int(os.getenv('NUMA_NODES').split(',')[rank])
+                command = f'numactl --cpunodebind={numa} --membind={numa} ' + command
```

Comment:
Shall we leverage the `__get_arguments_from_env` function from `ib_loopback_performance.py`? It seems more robust when reading NUMA_NODES array.